### ISSUE TITLE
BUG - Error ``max_iter >= 1`` when running ``skglm`` solver

### ISSUE THREAD

**Issue by Badr-MOUFAD** [benchopt/benchmark_elastic_net#10]
I think this has to do with the new release of sklearn ``v1.1.0``.

to reproduce 

```shell
benchopt run . -s skglm -d Simulated
```

Logs

```shell
  File "C:\Users\HP\Desktop\benchmarks\benchmark_elastic_net\venv\lib\site-packages\sklearn\utils\validation.py", line 1476, in check_scalar
    raise ValueError(
ValueError: max_iter == 0, must be >= 1.
    |--skglm: error
```

**Comment by mathurinm** [comment]
You are right! 
see here for how to fix it fix: https://github.com/benchopt/benchmark_lasso/pull/81

**Comment by Badr-MOUFAD** [comment]
fixed by #8
